en/renew-tls-certificate.md

```diff
@@ -0,0 +1,221 @@
+
+### Renew and replace the CA certificate and certificates between components
+
+When using `cert-manager` to issue the certificate, the `cert-manager` can automatically update the certificate before it expires by specifying the `spec.renewBefore` of the `Certificate` resource.
+
+Although `cert-manager` can automatically renew the CA certificate and the corresponding Kubernetes Secret objects, it currently does not support merging the old and new CA certificates into a combined CA certificate to accept certificates issued by the new and old CA certificates at the same time. Therefore, when renewing and replacing the CA certificate, the issue that TLS between cluster components can not authenticate each other might occur.
+
+> **Warning:**
+>
+> Because the certificates issued by the new and old CAs can not be accepted between components at the same time, renewing and replacing certificates needs to rebuild the Pods of some components. This might cause some requests to access the TiDB cluster to fail.
+
+The steps to renew and replace the CA certificates of PD, TiKV, TiDB and certificates between components are as follows.
+
+1. The `cert-manager` automatically renews the CA certificate and the Kubernetes Secret object `${cluster_name}-ca-secret` before the certificate expires.
+
+    `${cluster_name}` is the name of the cluster.
+
+    To manually renew the CA certificate, you can directly delete the corresponding Kubernetes Secret objects and trigger the `cert-manager` to regenerate the CA certificate.
+
+2. Delete the Kubernetes Secret objects corresponding to the certificate of each component.
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl delete secret ${cluster_name}-pd-cluster-secret --namespace=${namespace}
+    kubectl delete secret ${cluster_name}-tikv-cluster-secret --namespace=${namespace}
+    kubectl delete secret ${cluster_name}-tidb-cluster-secret --namespace=${namespace}
+    kubectl delete secret ${cluster_name}-cluster-client-secret --namespace=${namespace}
+    ```
+
+    In the above command, `${cluster_name}` is the name of the cluster, and `${namespace}` is the namespace deployed for the TiDB cluster.
+
+3. Wait for the cert-manager to issue new certificates for each component based on the new CA certificate.
+
+    Observe the output of `kubectl get secret --namespace=${namespace}` until the Kubernetes Secret objects corresponding to all components are created.
+
+4. Rebuild the Pod of PD, TiKV and TiDB component in sequence forcibly.
+
+    Because the `cert-manager` does not support combined CA certificates, if you try to perform a rolling update of each component, the Pods using the new and old CAs to issue certificates can not communicate normally based on TLS. Therefore, you need to delete the Pod forcibly and rebuild the Pod based on the certificate issued by the new CA.
```

Comment:
```suggestion
    Because `cert-manager` does not support combined CA certificates, if you try to perform a rolling update of each component, the Pods using the different CAs to issue certificates cannot communicate with each other via TLS. Therefore, you need to delete the Pods forcibly and recreate the Pods based on the certificate issued by the new CA.
```